pkg/security/resolvers/process/resolver.go

```diff
@@ -1217,6 +1210,15 @@ func (p *Resolver) GetEntryCacheSize() float64 {
 	return float64(p.cacheSize.Load())
 }
 
+// GetEntryCache return the resolver's entrycache
+func (p *Resolver) GetEntryCache(entryToEvent func(_ *model.ProcessCacheEntry)) {
```

Comment:
Sorry, I didn't see that this function was existing already existing : https://github.com/DataDog/datadog-agent/blob/main/pkg/security/resolvers/process/resolver.go#L1226

Reply:
Wow ! I did not even see it